src/common/POLFields/DetailsFields/FieldOrderFormat.js

```diff
@@ -60,9 +62,12 @@ function FieldOrderFormat({
     });
   };
 
+  const isBindaryActive = get(formValues, 'details.isBindaryActive', false);
+  const dataOptions = isBindaryActive ? OPTIONS_WITH_BINDARY_ACTIVE : ORDER_FORMAT_OPTIONS;
```

Comment:
What would a user see as the selected option if `electronicResource` or `other` was selected before activating `isBindaryActive`?